apps/model/model.js

```diff
@@ -740,8 +740,8 @@ function uploadModel() {
 }
 
 async function deleteModel(name) {
-  modelName = name.split('/').pop().split('_').splice(2).join('_').slice(0, -3);
-  if (confirm('Are you sure you want to delete ' + modelName + ' model?')) {
+  DeletedmodelName = name.split('/').pop().split('_').splice(2).join('_').slice(0, -3);
```

Comment:
sorry to nitpick, but can we do camelCase?
```suggestion
  deletedModelName = name.split('/').pop().split('_').splice(2).join('_').slice(0, -3);
```
(Thanks for fixing this!)

Reply:
Thanks I will change all the instances of DeletedmodelName